packages/relay/src/lib/eth.ts

```diff
@@ -157,9 +160,38 @@ export class EthImpl implements Eth {
     this.chain = chain;
     this.precheck = new Precheck(mirrorNodeClient, nodeClient, logger, chain);
     this.cache = cache;
+    this.registry = register;
     if (!cache) this.cache = new LRU(this.options);
   }
 
+  /* Temporary code duplication for a temporary HotFix workaround while the definitive  fix is implemented
+  * Please remove initClient workaround, or better yet, do not let this be merged on MAIN, short-lived fix intended only for 0.23.1 version */
+  private requestsPerSdkClient = 0;
+  private maxRequestsPerSdkClient = process.env.ETH_CALL_MAX_REQUEST_PER_SDK_INSTANCE || 50;
+  private hederaNetwork: string = (process.env.HEDERA_NETWORK || '{}').toLowerCase();
+  private ethSdkClient: SDKClient | undefined;
```

Comment:
Actually, do we need this?
Maybe this is what @david-bakin-sl was referring to.
If `getSdkCLient()` is a true getter then the variable should be a local one and return in the function

Reply:
We need it to live outside since we want to reuse the instance for X amount of request.
David was referencing to rename it so we can be sure as reviewer that is no longer being used outside where we want to keep on using it. (only for unit tests) so it should be visible the changes only there and on the constructor when is being injected.